blocks/card/card.css

```diff
@@ -0,0 +1,145 @@
+.card{
+    padding:20px 10px 60px;
+}
+.card__items{
+    list-style: none;
+    padding: 0;
+}
+.card__card{
+    padding: 20px 0;
+}
+.card__image{
+    position: relative;
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-height:460px ;
+    border-radius: 20px;
+}
+.card__image_type_customers{
+    background-image: url(../../images/forCustomers_mobile.png);
+}
+.card__image_type_wedding{
+    background-image: url(../../images/weddings_mobile.png);
+}
+.card__image_type_group{
+    background-image: url(../../images/groupBook_mobile.png);
+}
+.card__image_type_corporate{
+    background-image: url(../../images/corporate_mobile.png);
+}
+.card__content{
+    position: absolute;
+    inset: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: rgba(0, 0, 0, .5);
+    height: 100%;
+    border-radius: 20px;
+}
+.card__heading{
+    color: #fff;
+    font-family: 'Roboto Condensed', Arial, Helvetica, sans-serif;
+    font-size: 34px;
+    font-weight: 700;
+    padding: 36px 16px;
+    margin: 0;
+}
+.card__sub-content{
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+    align-items: flex-start;
+    justify-content: center;
+}
+.card__paragraphs{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+.card__text{
+    color: #fff;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+    font-weight: 400;
+    line-height: 20px;
+    font-size: 16px;
+    max-width: 584px;
+    margin: 0;
+    padding-bottom: 40px;
+}
+.card__text_type_first{
+    padding-bottom: 20px;
+}
+.card__button{
+    display: block;
+    color: #683D83;
+    font-size: 24px;
+    font-family: 'Roboto Condensed', Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    line-height: 28px;
+    text-transform: uppercase;
+    padding: 16px 69px;
```

Comment:
Set `padding: 16px;` instead to prevent weird text wrapping. `width: 100%;` remove min-width as soon as page gets wider.